AirCastingTests/Utils/TestDefaultProviding.swift

```diff
@@ -0,0 +1,40 @@
+// Created by Lunar on 14/06/2021.
+//
+
+import Foundation
+
```

Comment:
LOL

Reply:
Each type has a specifying more random way to generate values. I would prefer You use `UUID().uuidString` for strings, `Double.random()` etc etc.

Reply:
This is an implementation of a `Dummy Value` test-pattern, not a `Generated Value` test pattern. This is ought to be used in places where the test don't care at all what value it gets. I think randomization for this case is suboptimal (can lead to flaky tests) and if needed should be done using another type of static pattern:

```Swift
extension Int: Randomizable {
   static var random: Self { ... } 
}
```